NAMESPACE

```diff
@@ -90,10 +91,26 @@ importFrom(rlang,"!!")
 importFrom(rlang,.data)
 importFrom(rlang,.env)
 importFrom(rlang,arg_match)
+importFrom(rlang,call2)
+importFrom(rlang,caller_arg)
+importFrom(rlang,caller_env)
+importFrom(rlang,check_dots_empty0)
 importFrom(rlang,enquo)
 importFrom(rlang,enquos)
+importFrom(rlang,eval_tidy)
+importFrom(rlang,f_env)
+importFrom(rlang,f_rhs)
+importFrom(rlang,is_environment)
+importFrom(rlang,is_formula)
+importFrom(rlang,is_function)
 importFrom(rlang,is_missing)
 importFrom(rlang,is_quosure)
+importFrom(rlang,is_string)
+importFrom(rlang,missing_arg)
+importFrom(rlang,new_function)
+importFrom(rlang,pairlist2)
+importFrom(rlang,quo_get_env)
+importFrom(rlang,quo_get_expr)
 importFrom(rlang,quo_is_missing)
```

Comment:
todo: update NEWS.md.  This should at least be pointed to under breaking changes due to breaking code using the `f = <function>` interface.

Reply:
Under the 0.6.0.9999 heading; this is the "development after 0.6.0" version.